Buildings/Experimental/DHC/Examples/Cooling/DistrictCooling.mo

```diff
@@ -0,0 +1,225 @@
+    annotation (Dialog(group="Control settings"));
+  parameter Modelica.Units.SI.Time tWai=30
+    "Waiting time"
+    annotation (Dialog(group="Control settings"));
+  // Pumps
+  parameter Buildings.Fluid.Movers.Data.Generic perCHWPum(
+    pressure=Buildings.Fluid.Movers.BaseClasses.Characteristics.flowParameters(
+      V_flow=mCHW_flow_nominal/1000*{0.2,0.6,0.8,1.0},
+      dp=(dpCHW_nominal+dpSetPoi+18000+30000)*{1.5,1.3,1.0,0.6}))
+    "Performance data for chilled water pumps"
+    annotation (Dialog(group="Pumps"));
+  parameter Buildings.Fluid.Movers.Data.Generic perCWPum(
+    pressure=Buildings.Fluid.Movers.BaseClasses.Characteristics.flowParameters(
+      V_flow=mCW_flow_nominal/1000*{0.2,0.6,1.0,1.2},
+      dp=(dpCW_nominal+60000+6000)*{1.2,1.1,1.0,0.6}))
+    "Performance data for condenser water pumps"
+    annotation (Dialog(group="Pumps"));
+  parameter Modelica.Units.SI.Pressure dpCHWPumVal_nominal=6000
+    "Nominal pressure drop of chilled water pump valve"
+    annotation (Dialog(group="Pumps"));
+  parameter Modelica.Units.SI.Pressure dpCWPumVal_nominal=6000
+    "Nominal pressure drop of chilled water pump valve"
+    annotation (Dialog(group="Pumps"));
+  // Network
+  parameter Integer nLoa=3
+    "Number of served loads"
+    annotation (Dialog(group="Network"));
+  parameter Modelica.Units.SI.MassFlowRate mLoa_flow_nominal=10
+    "Nominal mass flow rate in each load"
+    annotation (Dialog(group="Network"));
+  parameter Modelica.Units.SI.PressureDifference dpDis_sr_nominal=1500
+    "Nominal pressure drop for supply/return resistance"
+    annotation (Dialog(group="Network"));
+  final parameter Modelica.Units.SI.MassFlowRate mCon_flow_nominal[nLoa]=fill(mLoa_flow_nominal,nLoa)
+    "Nominal mass flow rate in each connection line";
+  final parameter Modelica.Units.SI.MassFlowRate m_flow_nominal=sum(
+    mCon_flow_nominal)
+    "Nominal mass flow rate in the distribution line";
+  final parameter Modelica.Units.SI.PressureDifference dp_nominal=sum(
+    dis.con.pipDisSup.dp_nominal)+sum(
+    dis.con.pipDisRet.dp_nominal)
+    "Nominal pressure drop in the distribution line";
+  // Buildings
+  parameter String filNam[nLoa]={
+    "modelica://Buildings/Resources/Data/Experimental/DHC/Loads/Examples/MediumOffice-90.1-2010-5A.mos",
+    "modelica://Buildings/Resources/Data/Experimental/DHC/Loads/Examples/MediumOffice-90.1-2010-5A.mos",
+    "modelica://Buildings/Resources/Data/Experimental/DHC/Loads/Examples/MediumOffice-90.1-2010-5A.mos"}
+    "Library path of the file with thermal loads as time series"
+    annotation (Dialog(group="Buildings"));
+  final parameter Modelica.Units.SI.HeatFlowRate QCoo_flow_nominal[nLoa](
+    max=-Modelica.Constants.eps)={Buildings.Experimental.DHC.Loads.BaseClasses.getPeakLoad(
+    string="#Peak space cooling load",
+    filNam=Modelica.Utilities.Files.loadResource(filNam[i])) for i in 1:nLoa}
+    "Design cooling heat flow rate (<=0)Nominal heat flow rate, negative";
```

Comment:
Please check the parameter description.

Reply:
Have changed to "Space cooling design load (<=0)".